src/main/java/org/jitsi/videobridge/IceTransport.java

```diff
@@ -196,10 +196,13 @@
         iceComponent = iceStream.getComponent(Component.RTP);
         iceStream.addPairChangeListener(iceStreamPairChangeListener);
 
+        //TODO(brian): this should be replaced by passing a log context
+        // when creating the child logger, but we don't have this
+        // value yet when creating the child logger, so we need to
+        // change the logger to allow adding context after creation,
+        // then we can get rid of the need for logPrefix here
         this.logPrefix
```

Comment:
Would it be terrible if we re-initialized the Logger instance when we create the Agent?

Reply:
yeah i don't think that would work.  i don't think it'll be too bad to enable adding context after a logger's initial creation though